db/repositories/__init__.py

```
@@ -0,0 +1,12 @@
+from db.repositories.run_repository import RunRepository, SQLiteRunRepository, create_sqlite_repository
+from db.repositories.profile_repository import ProfileRepository, SQLiteProfileRepository, create_sqlite_profile_repository
+
+__all__ = [
+    "RunRepository",
+    "SQLiteRunRepository", 
+    "create_sqlite_repository",
+    "ProfileRepository",
+    "SQLiteProfileRepository",
+    "create_sqlite_profile_repository",
+]
```

Ruff (0.14.6)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)